cmd/loadTests.go

```diff
@@ -88,37 +107,48 @@ type ErrorCount struct {
-	PipelineRunFailureCount           int64             `json:"runPipelineFailures"`
-	PipelineRunFailureRate            float64           `json:"runPipelineFailureRate"`
-	DeploymentFailureCount            int64             `json:"deploymentFailures"`
-	DeploymentFailureRate             float64           `json:"deploymentFailureRate"`
-	ErrorCounts                       []ErrorCount      `json:"errorCounts"`
-	Errors                            []ErrorOccurrence `json:"errors"`
-	ErrorsTotal                       int               `json:"errorsTotal"`
+	Timestamp                         string  `json:"timestamp"`
+	EndTimestamp                      string  `json:"endTimestamp"`
+	MachineName                       string  `json:"machineName"`
+	BinaryDetails                     string  `json:"binaryDetails"`
+	ComponentRepoUrl                  string  `json:"componentRepoUrl"`
+	NumberOfThreads                   int     `json:"threads"`
+	NumberOfUsersPerThread            int     `json:"usersPerThread"`
+	NumberOfUsers                     int     `json:"totalUsers"`
+	PipelineSkipInitialChecks         bool    `json:"pipelineSkipInitialChecks"`
+	LoadTestCompletionStatus          string  `json:"status"`
+	AverageTimeToSpinUpUsers          float64 `json:"createUserTimeAvg"`
+	MaxTimeToSpinUpUsers              float64 `json:"createUserTimeMax"`
+	AverageTimeToCreateResources      float64 `json:"createResourcesTimeAvg"`
+	MaxTimeToCreateResources          float64 `json:"createResourcesTimeMax"`
+	AverageTimeToRunPipelineSucceeded float64 `json:"runPipelineSucceededTimeAvg"`
+	MaxTimeToRunPipelineSucceeded     float64 `json:"runPipelineSucceededTimeMax"`
+	AverageTimeToRunPipelineFailed    float64 `json:"runPipelineFailedTimeAvg"`
+
+	AverageTimeToDeploymentSucceeded float64 `json:"deploymentSucceededTimeAvg"`
+	MaxTimeToDeploymentSucceeded     float64 `json:"deploymentSucceededTimeMax"`
+	AverageTimeToDeploymentFailed    float64 `json:"deploymentFailedTimeAvg"`
+
+	ItsAverageTimeToRunPipelineSucceeded float64 `json:"its_runPipelineSucceededTimeAvg"`
+	ItsMaxTimeToRunPipelineSucceeded     float64 `json:"its_runPipelineSucceededTimeMax"`
+	ItsAverageTimeToRunPipelineFailed    float64 `json:"its_runpipelineFailedTimeAvg"`
+
+	UserCreationFailureCount     int64   `json:"createUserFailures"`
+	UserCreationFailureRate      float64 `json:"createUserFailureRate"`
+	ResourceCreationFailureCount int64   `json:"createResourcesFailures"`
+	ResourceCreationFailureRate  float64 `json:"createResourcesFailureRate"`
+	PipelineRunFailureCount      int64   `json:"runPipelineFailures"`
+	PipelineRunFailureRate       float64 `json:"runPipelineFailureRate"`
+
+	DeploymentFailureCount int64   `json:"deploymentFailures"`
+	DeploymentFailureRate  float64 `json:"deploymentFailureRate"`
+
+	ItsPipelineRunFailureCount int64   `json:"its_runPipelineFailures"`
+	ItsPipelineRunFailureRate  float64 `json:"its_runPipelineFailureRate"`
```

Comment:
@naftalysh Please keep the naming consistent
```suggestion
	ItsPipelineRunFailureCount int64   `json:"itsRunPipelineFailures"`
	ItsPipelineRunFailureRate  float64 `json:"itsRunPipelineFailureRate"`
```

Reply:
Updated